ddl/ddl.go

```diff
@@ -147,6 +147,12 @@ type DDL interface {
 		info []*model.TableInfo,
 		onExist OnExist) error
 
+	// CreatePlacementPolicyWithInfo creates a placement policy
+	//
+	// WARNING: the DDL owns the `policy` after calling this function, and will modify its fields
```

Comment:
> WARNING: the DDL owns the `policy` after calling this function

What does 'own' mean exactly? Is it something like the 'ownership' in Rust?

Reply:
This comment is copied from `CreateTableWithInfo` .  But yes, I think it is something like 'ownership' in Rust, the policy in the argument will be referred by the job in args. I'm not sure what will happens if we use it after calling the function, but it's better not to do it.